src/measure_theory/lebesgue_measure.lean

```diff
@@ -259,6 +260,43 @@ instance locally_finite_volume : locally_finite_measure (volume : measure ℝ) :
   mem_nhds_sets is_open_Ioo ⟨sub_lt_self _ zero_lt_one, lt_add_of_pos_right _ zero_lt_one⟩,
   by simp only [real.volume_Ioo, ennreal.of_real_lt_top]⟩⟩
 
+instance sigma_finite_volume : sigma_finite (volume : measure ℝ) := by apply_instance
+
+lemma volume_Icc_pi {a b : ι → ℝ} : volume (Icc a b) = ∏ i, ennreal.of_real (b i - a i) :=
```

Comment:
My guess is that `ennreal.of_real (∏ i, b i - a i)` would be easier to work with. But don't change it if you have a reason for this form.

Reply:
These formulas are not equal, e.g, for `a = ![2, 2]`, `b = ![1, 1]`.

Reply:
Ah, good point.